deps/rabbit/test/feature_flags_SUITE.erl

```diff
@@ -112,59 +112,90 @@ end_per_suite(Config) ->
 init_per_group(feature_flags_v1, Config) ->
     rabbit_ct_helpers:set_config(Config, {enable_feature_flags_v2, false});
 init_per_group(feature_flags_v2, Config) ->
-    rabbit_ct_helpers:set_config(Config, {enable_feature_flags_v2, true});
+    %% Before we run `feature_flags_v2'-related tests, we must ensure that both
+    %% umbrellas support them. Otherwise there is no point in running them. The
+    %% `feature_flags_v1' group already covers testing in that case.
+
+    %% To determine that `feature_flags_v2' are supported, we can't
+    %% query RabbitMQ which is not started. Therefore, we check if the
+    %% source or bytecode of `rabbit_ff_controller' is present.
+
+    IsSupported = case rabbit_ct_helpers:get_config(Config, rabbitmq_run_cmd) of
+                      undefined ->
+                          %% make
+                          Dir1 = ?config(rabbit_srcdir, Config),
+                          File1 = filename:join([Dir1, "src", "rabbit_ff_controller.erl"]),
+                          SupportedPrimary = filelib:is_file(File1),
+                          case rabbit_ct_helpers:get_config(Config, secondary_umbrella) of
+                              undefined ->
+                                  [SupportedPrimary];
+                              _ ->
+                                  Dir2 = ?config(secondary_rabbit_srcdir, Config),
+                                  File2 = filename:join([Dir2, "src", "rabbit_ff_controller.erl"]),
+                                  SupportedSecondary = filelib:is_file(File2),
+                                  [SupportedPrimary, SupportedSecondary]
+                          end;
+                      _ ->
+                          %% bazel
+                          Dir1 = ?config(rabbit_srcdir, Config),
+                          File1 = filename:join([Dir1, "ebin", "rabbit_ff_controller.beam"]),
+                          SupportedPrimary = filelib:is_file(File1),
```

Comment:
This will work with Make as well.

Beside the paths to `File1` and `File2`, the rest of the code should be common to Make and Bazel. This should simplify it.